regulations/uitests/comment_test.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+
+from nose.tools import *  # noqa
+from selenium.webdriver.support.ui import WebDriverWait
+
+from regulations.uitests.base_test import BaseTest
+
+
+class CommentTest(BaseTest, unittest.TestCase):
+
+    job_name = 'Comment test'
+
+    def test_comment(self):
+        self.driver.get(self.test_url + '/preamble/2016_02749')
```

Comment:
Probably want to go to `../I` or similar to better recreate what we expect from users.